lib/pdk/cli/util/changelog_generator.rb

```diff
@@ -0,0 +1,58 @@
+module PDK
+  module CLI
+    module Util
+      class ChangelogGenerator
+        def generate_changelog
```

Comment:
Should be guarded here on whether the changelog rake task/gem is available

Reply:
Will add a check here

Reply:
As stated in the template here: https://github.com/puppetlabs/pdk-templates/blob/master/moduleroot/Rakefile.erb#L75 there is already a check in the Rakefile

Reply:
That assumes the person is using the pdk-template, or has not overridden it to not have the gem.

Reply:
Added exception if gem not installed